osquery/dispatcher/scheduled_tasks_queue.cpp

```diff
@@ -0,0 +1,70 @@
+
+/**
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under both the Apache 2.0 license (found in the
+ *  LICENSE file in the root directory of this source tree) and the GPLv2 (found
+ *  in the COPYING file in the root directory of this source tree).
+ *  You may select, at your option, one of the above-listed licenses.
+ */
+
+#include <algorithm>
+#include <limits>
+
+#include "osquery/dispatcher/scheduled_tasks_queue.h"
+
+namespace osquery {
+
+void ScheduledTaskQueue::add(TaskImplementation impl, UnixTime firstRunTime) {
```

Comment:
Should either of these be references?

Reply:
The first arg is lambda and it is better to just move in it. The second arg is a POD type (`unsigned long`) and it is cheaper to copy it. Does it make sense?